tasks/db_deploy/README.md

```diff
@@ -62,13 +62,13 @@ standards expected.
    migration from version x to version y. The `migrate_sql.py` file should have
    the SQL needed to perform the migration. See previous versions for examples
    on how the SQL should be formatted and expressed in functions. The unit test
-   files `test_migrate_vY.py` and `test_migrate_sql_vY.py` should contain unit
+   files `test_migrate_vy.py` and `test_migrate_sql_vy.py` should contain unit
    tests for all code written.
    ```bash
-   touch migrations/migrate_versions_4_to_5/migrate.py
-   touch migrations/migrate_versions_4_to_5/migrate_sql.py
-   touch test/unit_tests/migrations/migrate_versions_4_to_5/test_migrate_v5.py
-   touch test/unit_tests/migrations/migrate_versions_4_to_5/test_migrate_sql_v5.py
+   touch migrations/migrate_versions_x_to_y/migrate.py
```

Comment:
wonder if `_x_to_y` should be reverted back to `_4_to_5` per the information in step 1, as we are just showing an example.